extensions/objects/templates/list_of_sets_of_html_strings_editor.html

```diff
@@ -0,0 +1,8 @@
+<div class="container-fluid">
+  <div ng-repeat="choice in choices track by $index">
+    <select ng-model="selectedChoice" aria-label="choice">
+      <option ng-change="selections($index)" ng-repeat="index in allowedChoices()">{{index}}</option>
+    </select>
+    <span angular-html-bind="choice.id"></span>
```

Comment:
Hi @vibhor98, you might have to change this to use angular-html-bind as an element since its restrict is being changed to `E` in PR #5068. Thanks!

Reply:
Ah ok. I'll change it if your PR gets merged before mine as there would be travis failures other way around.